src/olympia/devhub/tasks.py

```diff
@@ -395,24 +408,47 @@ def run_validator(path, for_appversions=None, test_all_tiers=False,
         return json_result
 
 
-def track_validation_stats(json_result):
+def run_addons_linter(path, listed=True):
+    from .utils import fix_addons_linter_output
+    process = subprocess.Popen([
+        settings.ADDONS_LINTER_BIN,
+        path,
+        '--boring'
+    ], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    with statsd.timer('devhub.linter'):
+        stdout, stderr = process.communicate()
+
+    if stderr:
+        raise ValueError(stderr)
```

Comment:
the outer functions that call `run_addons_linter` catch exceptions and return a skeleton result that has an exception-message for the user in there. So it is being handled.